eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -254,8 +254,8 @@ stages:
                   Write-Host "Detected package name: $($detectedPackageName)"
                   if ($detectedPackageName -notmatch "-alpha")
                   {
-                    Write-Error "Found non alpha version artifact to publish as dev version. Failing publish step. VersionPolicy should be client or core in rush.json to get alpha version build."
-                    exit 1
+                    #Write-Error "Found non alpha version artifact to publish as dev version. Failing publish step. VersionPolicy should be client or core in rush.json to get alpha version build."
```

Comment:
why is this commented out?

Reply:
testing purposes, cleaning those out now